### DISCUSSION TITLE
moveTo function accepting multiple indices

### DISCUSSION THREAD

**Discussion by hondrox715** [fabricjs/fabric.js#8229] [Ideas]
I'm working on a project that has an HTML layers panel that matches the layers of objects on the canvas. Users will be able to drag and drop layers in the panel, including multiple layers at once, which in turn will change the order of the objects on the canvas. For this, I would need the moveTo function to accept an array of old indices and an array of new indices. Maybe there is already a similar method for accomplishing this?

**Comment by ShaMan123** [comment]
simply set `_objects`
